prompterator/main.py

```diff
@@ -468,14 +459,27 @@ def set_up_ui_labelling():
             if var != c.TEXT_ORIG_COL:
```

Comment:
Maybe I'd add these other columns as well:
```suggestion
            if var != c.TEXT_ORIG_COL and var not in c.COLS_NOT_FOR_PROMPT_INTERPOLATION:
```